myuw/templates/handlebars/card/international_student.html

```diff
@@ -4,8 +4,71 @@
         International Student
     </h3>
+    <div class="links-container">
+        <h4 class="myuw-card-heading">Visa and Immigration Rules</h4>
 
-    <div class="myuw-card-section last">
+        <ul class="unstyled-list">
+        {{#if seattle}}
+	    <li><a href="https://iss.washington.edu/travel-visas/" target="_blank" data-linklabel="Travel and visas for Int'l students">Travel and Visas</a></li>
+	    <li>
+		{{#if is_j1}}
+		    <a href="https://iss.washington.edu/regulations/j1/" target="_blank" data-linklabel="Visa and Immigration Rules">Visa and Immigration Rules for J-1 students</a>
+		{{else}}
+		    <a href="https://iss.washington.edu/regulations/f1/" target="_blank" data-linklabel="Visa and Immigration Rules">Visa and Immigration Rules for F-1 students</a>
+		{{/if}}
+	    </li>
+	    <li><a href="https://iss.washington.edu/new-students/faq/" target="_blank" data-linklabel="ISS: Frequently Asked Questions (FAQ)">Frequently Asked Questions (FAQ)</a></li>
+	    <li><a href="https://iss.washington.edu/employment/" target="_blank" data-linklabel="Int'l student employment">Employment and Internships</a></li>
+	    <li><a href="https://www.trumba.com/calendars/sea_enroll-iss" target="_blank" data-linklabel="Deadlines and Workshops Calendar">Deadlines and Workshops Calendar</a></li>
+	{{/if}}
+
+	{{#if bothell}}
+	    <li><a href="http://www.uwb.edu/cie/current-students/travel" target="_blank" data-linklabel="Travel and visas for Int'l students">Travel and Visas</a></li>
+	    <li><a href="https://www.uwb.edu/cie/current-students/immigration-documents" target="_blank" data-linklabel="Immigration Rules for F-1 students">Immigration Document for F-1 Students </a></li>
+	    <li><a href="https://www.uwb.edu/cie/current-students/vacation-quarter" target="_blank" data-linklabel="Vacation Quarter">Vacation Quarter</a></li>
+	    <li><a href="http://www.uwb.edu/cie/employment" target="_blank" data-linklabel="Int'l student employment">Employment and Internships</a></li>
+	    <li><a href="https://www.uwb.edu/cie/events" target="_blank" data-linklabel="Center for International Education Event Calendar">Center for International Education (CIE) Calendar</a></li>
+	{{/if}}
+
+	{{#if tacoma}}
+	    <li><a href="http://www.tacoma.uw.edu/iss/travel-visas" target="_blank" data-linklabel="Travel and visas for Int'l students">Travel and Visas</a></li>
+	    <li><a href="http://www.tacoma.uw.edu/iss/maintaining-f-1-status" target="_blank" data-linklabel="Rules to maintain F-1 status">Rules to maintain F-1 status</a></li>
+	    <li><a href="http://www.tacoma.uw.edu/isss/employment" target="_blank" data-linklabel="International Student Employment">International Student Employment</a></li>
+	    <li><a href="http://www.tacoma.uw.edu/international-student-services/events" target="_blank" data-linklabel="International Student and Scholar Services (ISSS) Calendar - Deadlines and Info Sessions">International Student and Scholar Services (ISSS) Calendar</a></li>
```

Comment:
This one is very long. It truncates in the Recently Visited at "International Student and Scholar..."

You might want to invert these so the differentiator portion is at the beginning: "ISSS Calendar (International Student and Scholar Services)" or "Calendar: ISSS"

Reply:
![myuw__home](https://user-images.githubusercontent.com/6953502/43481305-7189aff8-94ba-11e8-8338-4f05880cc434.png)

Reply:
change it to ISSS: Int'l...